torch/csrc/autograd/variable.h

```diff
@@ -92,16 +97,20 @@ struct VariableImpl : public at::TensorImpl {
 
 public:
   std::shared_ptr<Function> get_grad_accumulator();
+  virtual std::shared_ptr<Function>& get_grad_fn() {
```

Comment:
it's a bit annoying that now `grad_fn` lookups are more expensive than for all other attrs